Languages/JS/questions.json

```diff
@@ -81,6 +81,33 @@
 		"trueOrFalse": false,
 		"shortAns": "BOM stands for Browser Object Model. It provides interaction with the browser. The default object of a browser is a window.",
 		"tag": "JavaScript"
-		}
+	},
+	{
+		"id": 9,
+		"question": "What does being weakly typed mean for JavaScript?",
+		"details": "Strict typing is a method in programming languages that assigns each variable a data type that any assignment must adhere to. Weak typed languages, such as JavaScript, do not enforce any such rule. This means a variable can be assigned as an integer, and a string can be concatenated to it, casting the entire thing to a string.",
+		"questionType": "multi",
+		"trueOrFalse": false,
+		"shortAns": "Weakly typed means that you can assign a variable without delclaring what data type the variable uses.",
+		"tag": "JavaScript"
+	},
+	{
+		"id": 10,
+		"question": "What does the === operator mean?",
+		"details": "As JavaScript is weakly typed, variables can have the same value without being the same type. For example, 10 would be equal to '10'. Using three equal signs equates both the value and the type without conversion. As the second value in our example is a string, it would fail to match against the integer.",
+		"questionType": "multi",
+		"trueOrFalse": false,
+		"shortAns": "That either side of the operater are of same value and type without conversion.",
```

Comment:
Please format all shortAns in the syntax as " It is _____ " and do not use the same wording used in the question so that when this is added to a bunch of other short answers, the user wont immediately know which one is the correct answer.

Reply:
Apologies, I have updated the formatting: [f62de69](https://github.com/Tc2r1/Hacktoberfest_Interview_App/pull/79/commits/f62de6901b54495875aba00059601d87fcb95ddf)